src/main/scala/nl/knaw/dans/easy/stage/lib/JSON.scala

```diff
@@ -69,17 +68,15 @@ object JSON {
         ("predicate" -> HAS_PID) ~ ("object" -> s.urn) ~ ("isLiteral" -> true),
         ("predicate" -> HAS_MODEL) ~ ("object" -> "info:fedora/dans-model:recursive-item-v1"),
         ("predicate" -> HAS_MODEL) ~ ("object" -> "info:fedora/easy-model:EDM1DATASET"),
-        ("predicate" -> HAS_MODEL) ~ ("object" -> "info:fedora/easy-model:oai-item1"),
-        ("predicate" -> STORED_IN_DARKARCHIVE) ~ ("object" -> "true") ~ ("isLiteral" -> true)
```

Comment:
`STORED_IN_DARKARCHIVE` is no longer necessary. The `dsLocation`s indicate where the files are stored.